frontend/src/views/dev_frame/load-mock-data.js

```diff
@@ -0,0 +1,227 @@
+                    data: mockPositionData,
+                },
+                positionSchema
+            ).map((p) =>
+                prepareFull.position(p, {
+                    instructors,
+                    contractTemplates,
+                })
+            );
+            for (let p of data) {
+                const position = await dispatch(upsertPosition(p));
+                positions.push(position);
+                count++;
+                setNow(Math.round((count / total) * 100));
+            }
+        } catch (error) {
+            console.error(error);
+            return error;
+        }
+    }
+
+    async function setupMockApplicants() {
+        try {
+            setStage("Applicants");
+            setNow(0);
+            count = 0;
+            total = mockApplicantData.length;
+            const data = normalizeImport(
+                {
+                    fileType: "json",
+                    data: mockApplicantData,
+                },
+                applicantSchema
+            );
+            for (let a of data) {
+                const applicant = await dispatch(upsertApplicant(a));
+                applicants.push(applicant);
+                count++;
+                setNow(Math.round((count / total) * 100));
+            }
+        } catch (error) {
+            console.error(error);
+            return error;
+        }
+    }
+
+    async function setupMockAssignments() {
+        try {
+            setStage("Assignments");
+            setNow(0);
+            count = 0;
+            total = mockAssignmentData.length;
+            const data = normalizeImport(
+                {
+                    fileType: "json",
+                    data: mockAssignmentData,
+                },
+                assignmentSchema
+            ).map((p) =>
+                prepareFull.assignment(p, {
+                    positions,
+                    applicants,
+                    session,
+                })
+            );
+            for (let a of data) {
+                await dispatch(upsertAssignment(a));
+                count++;
+                setNow(Math.round((count / total) * 100));
+            }
+        } catch (error) {
+            console.error(error);
+            return error;
+        }
+    }
+
+    async function clickHandler() {
+        try {
+            const sessions = await dispatch(fetchSessions());
+            if (sessions.some((s) => s.name === mockSessionData.name)) {
+                setDisable(true);
+                return;
+            }
+            setInProgress(true);
+            await setupMockSession();
+            await setupMockPositions();
+            await setupMockApplicants();
+            await setupMockAssignments();
+            setInProgress(false);
+        } catch (error) {
+            setInProgress(false);
+            return;
+        }
+    }
+
+    return (
+        <span
+            title={"One-click loads all mock data JSON files"}
```

Comment:
String attributes can be set directly

Reply:
This is supposed to be the tooltip content, as I tried to be consistent with the other two components, `ActiveUserButton` and `ToggleMockApi`.

Reply:
The tooltip content might need to be changed tho

Reply:
When it changes, then put the brackets back.